src/mots/module.py

```diff
@@ -0,0 +1,197 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+"""Module operations and utility functions."""
+
+from __future__ import annotations
+import logging
+from pathlib import Path
+import pprint
+
+logger = logging.getLogger(__name__)
+
+print = pprint.PrettyPrinter().pprint
```

Comment:
Nit:
```
from pprint import pprint
...
print = pprint
```
That should be functionally equivalent to `pprint.PrettyPrinter().pprint`, right?

Reply:
I suppose we could even do

```python
from pprint import pprint as print
```